### PR TITLE
Adding italian lanuguages and i18n fixes

### PR DESCRIPTION
**Pull request by mlgtcode**
* adding italian translation
* fixing translation errors in german translation
* adding missing iso language codes
* fix padding at footer and make the lang menu usable on mobile devices

### CODE REVIEW

File: src/locale/index.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { translation_de } from "./de";
 import { translation_en } from "./en";
+import { translation_de } from "./it";
```
Comment (MultiMote) [inline src/locale/index.ts:3]: Should be `import { translation_it } from "./it";`

**Review by MultiMote — CHANGES_REQUESTED** [review]
Thanks!
One fix is needed.
Please run `yarn sv-check` before creating pull request 🙂

**Comment by mlgtcode** [comment]
> Please run `yarn sv-check` before creating pull request 🙂

Will setup a development environment asap :)

**Review by MultiMote — APPROVED** [review]

**Comment by MultiMote** [comment]
@mlgtcode Language switcher is now broken on mobile.

https://github.com/user-attachments/assets/e69565a4-77ed-496b-ac24-dc4ca3f2cb36

**Pull request merged**